WebApp/processing/processing.go

```diff
@@ -24,8 +25,161 @@ func absDiff[T int | int8 | int16 | int32 | int64 | uint | uint8 | uint16 | uint
 	return b - a
 }
 
+func manhattanDistance(a Vertex, b Vertex) int {
+	return absDiff(int(a.X), int(b.X)) + absDiff(int(a.Y), int(b.Y))
+}
```

Comment:
Taking in two vertexes and returning the total absolute distance between their x and y's looks good.